htdocs/fourn/paiement/list.php

```diff
@@ -254,14 +259,11 @@
 if ($search_ref) {
 	$param .= '&search_ref='.urlencode($search_ref);
 }
-if ($saerch_day) {
-	$param .= '&search_day='.urlencode($search_day);
-}
-if ($saerch_month) {
-	$param .= '&search_month='.urlencode($search_month);
+if ($search_date_start) {
```

Comment:
We must always keep the detail of the 3 component (day, month, year) for search component so we shoul dhave
if ($search_date_startday) {

if ($search_date_startmonth) {
..